iac-extensions/arm/src/test/resources/checks/StringLiteralDuplicatedCheck/StringLiteralDuplicatedCheck.bicep

```diff
@@ -54,3 +54,40 @@ resource storageAccount 'Microsoft.Storage/storageAccounts' = {
   }
 }
 
+// Compliant: module paths cannot be variables
+module moduleName_one '../dosomething.bicep' = {
+  name: 'deployName_one'
+  params: {
+    argument: 'test1'
+  }
+}
+module moduleName_two '../dosomething.bicep' = {
+  name: 'deployName_two'
+  params: {
+    argument: 'test2'
+  }
+}
+module moduleName_tree '../dosomething.bicep' = {
+  name: 'deployName_tree'
+  params: {
+    argument: 'test3'
+  }
+}
+module moduleName_four '../dosomething.bicep' = {
+  name: 'deployName_four'
+  params: {
+    argument: 'test4'
+  }
+}
+module moduleName_five '../dosomething.bicep' = {
+  name: 'deployName_five'
+  params: {
+    argument: 'test5'
+  }
+}
+module moduleName_five '../dosomething.bicep' = {
```

Comment:
Please add a separate example for `dosomething.bicep ` and `dir/something.bicep`